src/mappening/api/events.py

```diff
@@ -0,0 +1,282 @@
+* ticketing: A JSON with a single ticket_uri element with a url to the ticketing site or <NONE>
+* end_time: String end time of event in the format Sat, 17 Feb 2018 23:30:00 GMT-0800
+* free_food: If event has free food, currently just a strong NO
+
+**Sample Event**::
+
+    {
+      "geometry": {
+        "coordinates": [
+          -118.451994,
+          34.071474
+        ],
+        "type": "Point"
+      },
+      "id": "1766863560001661",
+      "properties": {
+        "category": "<NONE>",
+        "cover_picture": "https://scontent.xx.fbcdn.net/v/t31.0-8/s720x720/27356375_1972757046097696_6206118120755555565_o.jpg?oh=2240b43f536e76f9cf00410f602af386&oe=5B136061",
+        "description": "Hack on the Hill IV (HOTH) is a 12 hour, beginner-friendly hackathon designed to give beginners a glimpse into what a real hackathon would be and feel like. During HOTH, there are workshops, mentors, and amazing prizes for the best hacks. As a sequel to HOTH III, HOTH IV features double the attendance and hacking tracks hosted by different ACM committees. We are also excited to announce that we'll be providing select hardware for hacking as well! LEARN MORE AND SIGN-UP HERE (applications close 2/10 at midnight): https://hoth.splashthat.com/ Sponsored by IS Associates, a UCLA-sponsored organization that provides an educational forum for the management and understanding of information technology. Learn more at: https://isassociates.ucla.edu",
+        "duplicate_occurrence": "NO",
+        "end_time": "Sat, 17 Feb 2018 23:30:00 GMT-0800",
+        "event_name": "ACM Hack | Hack on the Hill IV",
+        "free_food": "NO",
+        "hoster": {
+          "id": "369769286554402",
+          "name": "UCLA Class of 2020"
+        },
+        "is_cancelled": false,
+        "start_time": "Sat, 17 Feb 2018 08:30:00 GMT-0800",
+        "stats": {
+          "attending": 97,
+          "interested": 199,
+          "maybe": 199,
+          "noreply": 107
+        },
+        "ticketing": {
+          "ticket_uri": "https://hoth.splashthat.com/"
+        },
+        "venue": {
+          "id": "955967887795957",
+          "location": {
+            "city": "Los Angeles",
+            "country": "United States",
+            "latitude": 34.071474,
+            "longitude": -118.451994,
+            "state": "CA",
+            "street": "330 De Neve Dr Ste L-16",
+            "zip": "90024"
+          },
+          "name": "Carnesale Commons"
+        }
+      },
+      "type": "Feature"
+    }
+
+-----------------
+API DOCS
+-----------------
+"""
+# Interacting with events collection in mlab
+
+from mappening.utils.database import *
+from mappening.api.utils import event_caller, event_utils
+
+from flask import Flask, jsonify, request, json, Blueprint
+from flask_cors import CORS, cross_origin
+import requests, urllib
+import time, datetime, dateutil.parser
+import json
+import re
+from tqdm import tqdm
+
+events = Blueprint('events', __name__)
+
```

Comment:
Maybe put a comment here about the prefixing when you register the blueprint, so less experienced people dont get confused by short route